packages/flutter_tools/lib/src/platform_plugins.dart

```diff
@@ -109,6 +126,9 @@ class AndroidPlugin extends PluginPlatform implements NativeOrDartPlugin {
   /// The Dart plugin main class defined in pubspec.yaml, if any.
   final String? dartPluginClass;
 
+  /// Is FFI plugin defined in pubspec.yaml, if any.
+  final bool? ffiPlugin;
```

Comment:
Does this actually need to be nullable? Not present should be equivalent to `false`.

(Same in the other classes below.)

Reply:
Done.

Please note that `toMap()` will now not get an entry if `ffiPlugin: false`.